### PR TITLE
#159965308 Mockup for comment on article 

### PR DESCRIPTION
**Pull request by pabishai**
#### What does this PR do?
- Enable users to comment on articles and view comments on articles

#### Description of the task to be completed?
- User should be able to post a comment on an article and view comments for an article

#### How should this be manually tested?
```
$ git clone https://github.com/andela/ah-magnificent6.git
$ git checkout ft-mockup-comment-article-159965308
$ cd ah-magnificent6/ui-mockups
$ brew install node
$ npm install
$ npm install gulp-cli -g
$ npm install gulp -D
$ gulp
For search navigate to http://localhost:3000/article.html
```

##### Screenshots
screen shot 2018-09-25 at 00 06 35

### CODE REVIEW

File: ui-mockups/src/article.html [hunk 1]
```diff
@@ -171,8 +171,28 @@ <h1 class="card-title">Article title: My journey into Andela</h1>
...
+                            <p>Start a discussion not a fire. Post with kindness</p>
+                            <textarea rows="2" cols="100" name="comment" form="usrform">Post a comment</textarea>
+                        <div class="col col-md-1">
+                            <!-- Usr icon -->
+                            <div class="thumbnail">
+                                    <i class="fas fa-user-circle"></i>
+                            </div>
+                        </div>
+                        <div class="col col-md-11">
+                            <div class="panel panel-default">
```
Comment (pngg) [inline ui-mockups/src/article.html]: Are we using Bootstrap 3 here? We don't have `panel` and `thumbnail` in Bootstrap 4 but instead use `cards`.

**Review by pngg — CHANGES_REQUESTED** [review]

File: ui-mockups/src/article.html (hunk 1 above)
Comment (meshack-mbuvi) [reply ui-mockups/src/article.html]: I agree with you @pngg

**Comment by meshack-mbuvi** [comment]
@PAbishai Looks good to me

File: ui-mockups/src/article.html (hunk 1 above)
Comment (pabishai) [reply ui-mockups/src/article.html]: Noted @pngg . I thought it didn't matter due to backward compatibility but I get your concern. Let me work on it.

Comment (pabishai) [reply ui-mockups/src/article.html]: @pngg The changes have been effected.

**Review by pngg — APPROVED** [review]

**Pull request merged**